modules/nodes-containers-downward-api-container-secrets.adoc

```diff
@@ -53,15 +57,18 @@ spec:
               name: mysecret
               key: username
   restartPolicy: Never
+# ...
 ----
 
-. Create the pod from the `pod.yaml` file:
+.. Create the pod from the `pod.yaml` file:
 +
 [source,terminal]
 ----
 $ oc create -f pod.yaml
 ----
 
+.Verification
```

Comment:
I don't think this should have a number, if it's the only verification criteria. See RH supplementary style guide: https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures